frontend/src/Pages/Admins.tsx

```diff
@@ -175,7 +175,6 @@ export default function Users() {
                                 <span>Name</span>
                             </th>
                             <th>Username</th>
-                            <th>Role</th>
```

Comment:
While you removed the header, the information is still being rendered in the table (see screenshot inserted below). Make sure you remove the entire column- applies to both pages.
<img width="1508" alt="Screenshot 2024-09-25 at 12 31 43 PM" src="https://github.com/user-attachments/assets/05279ec5-5795-48e4-b898-c4a753902af9">